cryptoki/tests/basic.rs

```diff
@@ -826,3 +826,46 @@ fn aes_cbc_pad_encrypt() -> TestResult {
     assert_eq!(expected_cipher[..], cipher[..]);
     Ok(())
 }
+
+#[test]
+#[serial]
+fn update_attributes_key() -> Result<()> {
```

Comment:
This should be `TestResult` now I think.

Reply:
sorry, missed that.